netlify.toml

```diff
@@ -0,0 +1,5 @@
+[build]
+  publish = ".next"
+
+[[plugins]]
+  package = "@netlify/plugin-nextjs"
```

Comment:
Missing newline at EOF over here. It's a trivial thing but it can possibly break and/or introduce hard to identify bugs. So add a newline at EOF.